### ISSUE TITLE
Error: Extension host terminated unexpectedly

### ISSUE THREAD

**Issue by SebastianKristof** [jaime-olivares/vscode-yuml#55]
I was viewing a diagram, then I closed the preview window. Then I tried to open preview again by hitting a button in the top right corner, but the preview didn't appear. Closing and reopening the .yuml file did not help. After a few seconds, the error message appeared.

VS Code Version 1.30.2 (1.30.2), Mac

```
extensionService.ts:165 Extension host terminated unexpectedly. Code:  7  Signal:  null
t._onExtensionHostCrashed @ extensionService.ts:165
```
```
extensionHost.ts:236 /Users/cem/.vscode/extensions/jaimeolivares.yuml-3.1.2/node_modules/viz.js/viz.js:25
var Module=typeof Module!=="undefined"?Module:{};var moduleOverrides={};var key;for(key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}Module["arguments"]=[];Module["thisProgram"]="./this.program";Module["quit"]=(function(status,toThrow){throw toThrow});Module["preRun"]=[];Module["postRun"]=[];var ENVIRONMENT_IS_WEB=false;var ENVIRONMENT_IS_WORKER=false;var ENVIRONMENT_IS_NODE=false;var ENVIRONMENT_IS_SHELL=false;if(Module["ENVIRONMENT"]){if(Module["ENVIRONMENT"]==="WEB"){ENVIRONMENT_IS_WEB=true}else if(Module["ENVIRONMENT"]==="WORKER"){ENVIRONMENT_IS_WORKER=true}else if(Module["ENVIRONMENT"]==="NODE"){ENVIRONMENT_IS_NODE=true}else if(Module["ENVIRONMENT"]==="SHELL"){ENVIRONMENT_IS_SHELL=true}else{throw new Error("Module['ENVIRONMENT'] value is not valid. must be one of: WEB|WORKER|NODE|SHELL.")}}else{ENVIRONMENT_IS_WEB=typeof window==="object";ENVIRONMENT_IS_WORKER=typeof importScripts==="funct

Error: write EPIPE
	at _errnoException (util.js:1024:11)
	at WriteWrap.afterWrite [as oncomplete] (net.js:867:14)
```

**Comment by jaime-olivares** [comment]
I was just working on a new presentation logic, so I will publish soon and let's see if this issue persists.

**Comment by jaime-olivares** [comment]
Published 3.2. Please verify if the issue has gone.

**Comment by SebastianKristof** [comment]
So far it's working... will keep an eye on it for a few days. Thanks for the tool btw!
